cypress/integration/models/jobfunctions.ts

```diff
@@ -24,48 +27,48 @@ export class Jobfunctions {
         inputText(jobfunctionNameInput, name);
     }
 
-    getJobFuncName(): string {
-        this.jobfunctionName = faker.name.jobType();
-        return this.jobfunctionName;
-    }
-
-    create(): void {
+    create(cancel = false): void {
         Jobfunctions.clickJobfunctions();
-        clickByText("button", "Create new");
-        this.getJobFuncName();
-        this.fillName(this.jobfunctionName);
+        clickByText(button, createNewButton);
+        this.fillName(this.name);
         submitForm();
     }
 
-    edit(): void {
+    edit(updatedName: string, cancel = false): void {
         Jobfunctions.clickJobfunctions();
```

Comment:
can we keep it as `name` only.

Reply:
No, it has to be named correctly, we are providing updatedName. using name is ambigous as it is the class member.